### PR TITLE
Preparing external_vpn_gateway example for inclusion on c.g.c.

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
The intention is to add this example to https://cloud.devsite.corp.google.com/network-connectivity/docs/vpn/how-to/automate-vpn-setup-with-terraform.

```release-note:none
```

### CODE REVIEW

File: mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb [hunk 1]
```diff
@@ -99,3 +100,4 @@ resource "google_compute_router_peer" "router1_peer2" {
   advertised_route_priority = 100
   interface                 = google_compute_router_interface.router1_interface2.name
 }
+# [START cloudvpn_ha_external]
```
Comment (ScottSuarez) [inline mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb:103]: What are these START and START identifiers ?

Should this be START and START enclosing this?

**Review by ScottSuarez — COMMENTED** [review]
small question

File: mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb (hunk 1 above)
Comment (betsy-lichtenberg) [reply mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb:103]: Good catch It should be START and END. These are region tags, which tell DevSite which part of the code to include on the website. In this case, we want to include all of it, so I have enclosed all of it with these tags. My next commit will fix the closing tag.

Comment (betsy-lichtenberg) [reply mmv1/templates/terraform/examples/external_vpn_gateway.tf.erb]: By the way, the expert on this workflow is camthornton@ (on github, he is c2thorn).

**Review by ScottSuarez — COMMENTED** [review]
lgtm !

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**